src/audit.js

```diff
@@ -24,12 +24,12 @@ export const playAudit = async function (auditConfig = {}) {
   if (process.env.LIGHTHOUSE_LAMBDATEST === 'true') {
     if (!auditConfig.page) {
       throw new Error(
-        `Please pass the page object to generate lighthouse report on LambdaTest.`
+        `Please pass the page object to generate lighthouse report on LambdaTest.`,
```

Comment:
I think the dangling comma comes from the prettier upgrade - Can be turned off if preferred.